src/lib/openapi/spec/create-strategy-variant-schema.ts

```diff
@@ -0,0 +1,55 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const createStrategyVariantSchema = {
+    $id: '#/components/schemas/createStrategyVariantSchema',
+    type: 'object',
+    additionalProperties: true,
```

Comment:
The default value of this property is `true`, so you don't strictly **need** this line, but you can leave it in if you think it makes sense.

```suggestion
```

Reply:
I prefer the explicit version since I never remember the defaults